lib/policies/jwt/index.js

```diff
@@ -38,6 +38,10 @@ module.exports = {
         type: 'boolean',
         default: true,
         description: 'Value istructing the gateway whether verify the sub against the internal SOC'
+      },
+      algorithms: {
+        type: 'array',
```

Comment:
I think we should probably specify the `items` to be a string and then enum the possible choice. JWT has a fixed set of algorithms

Reply:
That makes sense, I just pushed up a change to reflect that. I wasn't 100% sure about the syntax so let me know if I need to modify it at all.